### PR TITLE
Replace many getProperty with getIceProperty

### PR DESCRIPTION
**Pull request by externl**
This PR updates many of the places we get Ice properties using `getProperty` to use `getIceProperty`.

### CODE REVIEW

**Review by bernardnormier — APPROVED** [review]

File: java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/Coordinator.java
Comment (pepone) [inline java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/Coordinator.java:1]: Some changes in this file seem unrelated to the PR.

**Review by pepone — APPROVED** [review]
